R/endpoints.R

```diff
@@ -863,22 +924,29 @@ ecdc_ili <- function(regions,
   )
 }
 
-#' fetch FluSurv virological data
+#' Fetch FluSurv virological data
 #'
 #' Obtain information on flu hospitalization rates from the Center of Disease Control.
 #'
-#' @details The list of location argument can be found in <https://github.com/cmu-delphi/delphi-epidata/blob/main/labels/flusurv_locations.txt>.
+#' @details The list of location argument can be found in
+#' <https://github.com/cmu-delphi/delphi-epidata/blob/main/labels/flusurv_locations.txt>.
 #'
 #' @examples
+#' \donttest{
 #' call <- flusurv(locations = "CA", epiweeks = epirange(201201, 202001))
 #' fetch_classic(call)
-#' @param locations A character string indicating location.
-#' @param epiweeks Takes in the form epirange(startdate,enddate), where startdate and enddate are of the form YYYYMMDD (can be passed as string or numeric).
-#' @param issues optionally specify the exact issues to fetch
-#' @param lag optionally specify the issue lag
-#' @return an instance of epidata_call
-#'
-#' @references API Documentation: <https://cmu-delphi.github.io/delphi-epidata/api/flusurv.html>.   See also <https://gis.cdc.gov/GRASP/Fluview/FluHospRates.html>.
+#' }
+#' @param locations Character vector. Character strings indicating location.
+#' @param epiweeks Epirange. The epiweeks to fetch the form epirange(startweek,endweek), where startweek
+#'   and endweek are of the form YYYYWW (string or numeric).
+#' @param issues Epirange. Optionally, specify the issues to fetch in the form epirange(startweek,endweek), where
+#'   startweek and endweek are of the form YYYYWW (string or numeric). Mutually exclusive with `lag`.
+#' @param lag Integer. Optionally, specify the lag for the issues to fetch. An alternative to and mutually exclusive
+#'   with `issues`.
+#' @return An instance of epidata_call
```

Comment:
praise: really liking the way param and return types are noted.

suggestion: ``[`epidata_call`]``... and make that actually point to a help topic.